base/indices.jl

```diff
@@ -364,14 +364,14 @@ IndexStyle(::Type{<:LinearIndices}) = IndexLinear()
 axes(iter::LinearIndices) = map(indices1, iter.indices)
 size(iter::LinearIndices) = map(unsafe_length, iter.indices)
 function getindex(iter::LinearIndices, i::Int)
-    @_inline_meta
+    @_propagate_inbounds_meta
```

Comment:
Propagate inbounds should only be needed if this function itself calls `getindex` — I think `@inline` should be correct here.  Did you see this make a difference?

Reply:
Ah, you're right, I am rusty on that machinery. Will correct. Thanks for looking!